src/System.Private.CoreLib/ILLinkTrim.xml

```diff
@@ -45,16 +45,32 @@
       <!-- Methods is used by VS Tasks Window. -->
       <method name="GetActiveTaskFromId" />
     </type>
+    <type fullname="System.Collections.Generic.GenericArraySortHelper`1">
+      <!-- Instantiated via reflection -->
```

Comment:
Just a comment for the future, it'd be nice if the XML format itself included a reason/justification property so these notes could be included in the XML rather than left as comments.  Obviously not a big deal, though.

Reply:
Just curious - why do you think attribute would be better?

I actually like comments better because of it makes it is same style as code comments.

Reply:
Personally I prefer the attribute because it prescribes a consistent way to include the reasoning, could even be made mandatory by the tool, and could allow the reasoning to be included in a report or the like if that was deemed valuable (much of that could of course be done with comments as well, but by their very nature comments are less strict).  I realize this is likely a case of personal preference and there's no "right" answer.